tests/containers/bci_prepare.pm

```diff
@@ -106,6 +106,14 @@ sub run {
         record_info('IMAGE', $image);
         script_retry("$engine pull $image", timeout => 300, delay => 60, retry => 3);
         record_info('Inspect', script_output("$engine inspect $image"));
+
+        my $build = get_var('CONTAINER_IMAGE_BUILD');
+        if ($build && $build ne 'UNKNOWN') {
+            record_info('CONTAINER_IMAGE_BUILD', $build);
+            my $reference = script_output(qq($engine inspect --type image $image | jq -r '.[0].Config.Labels."org.opensuse.reference"'));
+            record_info('org.opensuse.reference', $reference);
```

Comment:
Similar here, 
![image](https://user-images.githubusercontent.com/11728254/173809886-81080e15-ef6a-48f7-a815-e757834bb059.png)

but however, this `record_info` is useful.

```suggestion
            record_info('Image ref', "org.opensuse.reference = $reference");
```

Reply:
I did:
```suggestion
            record_info('image ref', "org.opensuse.reference: $reference");
```